### PR TITLE
Fusionstorage driver support for Fusionstorage storage device version 8.0 

### PR DESCRIPTION
**Pull request by jackhaibo**
**What this PR does / why we need it**:
This pr implements fusionstorage driver in hotpot support for the fusionstorage storage device version 8.0.

**Special notes for your reviewer**:
@wisererik @leonwanghui please take a look at this pr.

### CODE REVIEW

File: contrib/connector/iscsi/helper.go [hunk 1]
```diff
@@ -236,7 +236,15 @@ func connect(connMap map[string]interface{}) (string, error) {
 	log.Println("connmap info: ", connMap)
 	log.Println("conn info is: ", conn)
 	portal := conn.TgtPortal[index]
-	targetiqn := conn.TgtIQN[index]
+
+	var targetiqn string
+	if len(conn.TgtIQN) == 0 {
+		content, _ := discovery(portal)
+		targetiqn = strings.Split(content, " ")[1]
```
Comment (wisererik) [inline contrib/connector/iscsi/helper.go]: define constant for  magic number

File: contrib/connector/iscsi/helper.go [hunk 2]
```diff
@@ -262,7 +270,7 @@ func connect(connMap map[string]interface{}) (string, error) {
 	log.Println("devicepath is ", devicePath)
 
 	// Discovery
-	err = discovery(portal)
+	_, err = discovery(portal)
 	if err != nil {
 		return "", err
 	}
```
Comment (wisererik) [inline contrib/connector/iscsi/helper.go]: return nil, err

File: contrib/drivers/huawei/fusionstorage/fsclient.go [hunk 3]
```diff
@@ -180,6 +179,8 @@ func (c *FsCli) createVolume(volName, poolId string, volSize int64) error {
 	url := "/volume/create"
 	polID, _ := strconv.Atoi(poolId)
 	params := map[string]interface{}{"volName": volName, "volSize": volSize, "poolId": polID}
+	fmt.Println(params)
+
```
Comment (wisererik) [inline contrib/drivers/huawei/fusionstorage/fsclient.go]: remove it

File: contrib/connector/iscsi/helper.go (hunk 2 above)
Comment (leonwanghui) [reply contrib/connector/iscsi/helper.go]: It should be "", if the return type is string.

File: contrib/drivers/huawei/fusionstorage/fsclient.go (hunk 3 above)
Comment (jackhaibo) [reply contrib/drivers/huawei/fusionstorage/fsclient.go]: Done.

**Comment by jackhaibo** [comment]
> previous
Test passed, while supporting versions 6.3 and 8.0.

**Comment by leonwanghui** [comment]
Please update your code to latest version.

**Pull request merged**